GitUI/UserControls/FileStatusList.cs

```diff
@@ -1411,18 +1416,18 @@ private void FileStatusListView_DrawSubItem(object sender, DrawListViewSubItemEv
 
                 if (!string.IsNullOrEmpty(prefix))
                 {
-                    DrawString(textRect, prefix, SystemColors.GrayText);
+                    DrawString(textRect, prefix, e.Item.Selected ? SystemColors.HighlightText : SystemColors.GrayText);
```

Comment:
Set textColor only once, use variable

Reply:
It won't work as expected because two different colors (`SystemColors.GrayText` and `SystemColors.ControlText`) are used for different parts of the text when the row is not selected.

Reply:
```
             var (image, prefix, text, suffix, prefixTextStartX, _, textMaxWidth) = FormatListViewItem(item, formatter, item.Bounds.Width);
 
-            if (e.Item.Selected)
+            if (item.Selected)
             {
                 e.Graphics.FillRectangle(SystemBrushes.Highlight, e.Bounds);
             }
@@ -1413,21 +1413,22 @@ private void FileStatusListView_DrawSubItem(object sender, DrawListViewSubItemEv
             if (!string.IsNullOrEmpty(text))
             {
                 var textRect = new Rectangle(prefixTextStartX, item.Bounds.Top, textMaxWidth, item.Bounds.Height);
+                var textColor = item.Selected ? SystemColors.HighlightText : SystemColors.GrayText;
 
                 if (!string.IsNullOrEmpty(prefix))
                 {
-                    DrawString(textRect, prefix, e.Item.Selected ? SystemColors.HighlightText : SystemColors.GrayText);
+                    DrawString(textRect, prefix, textColor);
                     var prefixSize = formatter.MeasureString(prefix);
                     textRect.Offset(prefixSize.Width, 0);
                 }
 
-                DrawString(textRect, text, e.Item.Selected ? SystemColors.HighlightText : SystemColors.ControlText);
+                DrawString(textRect, text, textColor);
 
                 if (!string.IsNullOrEmpty(suffix))
                 {
                     var textSize = formatter.MeasureString(text);
                     textRect.Offset(textSize.Width, 0);
-                    DrawString(textRect, suffix, e.Item.Selected ? SystemColors.HighlightText : SystemColors.GrayText);
+                    DrawString(textRect, suffix, textColor);
                 }
             }
 
```

Reply:
Please look closer to these lines:
```
DrawString(textRect, text, e.Item.Selected ? SystemColors.HighlightText : SystemColors.ControlText);
```
and
```
DrawString(textRect, suffix, e.Item.Selected ? SystemColors.HighlightText : SystemColors.GrayText);
```
Different colors are used for path and file name. It isn't introduced by me. These colors are used in the original code for not selected rows.

Reply:
thanks!